lms/static/scripts/frontend_apps/components/JSTORPicker.js

```diff
@@ -219,6 +227,39 @@ export default function JSTORPicker({ onCancel, onSelectURL }) {
             </div>
           )}
 
+          {metadata.data && (
+            <>
+              <div className="grow" />
+              <div className="flex space-x-2 items-center px-1">
+                <label htmlFor="accept-jstor-terms" className="grow text-right">
+                  I accept the{' '}
+                  <Link
+                    href="https://about.jstor.org/terms/"
+                    classes="underline hover:underline"
+                    target="_blank"
+                  >
+                    terms and conditions
+                  </Link>{' '}
+                  for JSTOR content use
+                </label>
+                <div className="text-lg">
+                  <Checkbox
+                    checked={acceptedTerms}
+                    data-testid="jstor-terms-checkbox"
+                    id="accept-jstor-terms"
+                    name="accept-terms"
+                    onInput={e =>
+                      setAcceptedTerms(
+                        /** @type {HTMLInputElement} */ (e.target).checked
+                      )
+                    }
+                    onKeyDown={onKeyDown}
```

Comment:
This is to ensure that Enter key presses submit the form while the checkbox is focused?